pyiron/lammps/interactive.py

```diff
@@ -563,23 +563,16 @@ def interactive_stress_getter(self):
             self._interactive_lib_command("compute st all stress/atom NULL")
             self._interactive_lib_command("run 0")
             self.interactive_cache["stress"] = []
-        ss = (
-            np.array(
-                [
-                    self._interactive_library.extract_compute("st", 1, 2)[i][
-                        j + (j != k) * (k + 2)
-                    ]
-                    for i in range(len(self.structure))
-                    for j in range(3)
-                    for k in range(3)
-                ]
-            ).reshape(len(self.structure), 3, 3)
-            / 1.602e6
-        )
+        ID = self._interactive_library.extract_atom("id", 0)
+        ID = np.array([ID[i] for i in range(len(self.structure))])-1
+        ID = np.arange(len(ID))[np.argsort(ID)]
+        ind = np.array([0, 3, 4, 3, 1, 5, 4, 5, 2])
+        ss = self._interactive_library.extract_compute("st", 1, 2)
+        ss = np.array([ss[i][j] for i in range(len(self.structure)) for j in range(6)]).reshape(-1, 6)[ID]
+        ss = ss[:, ind].reshape(len(self.structure), 3, 3)/1.602e6
```

Comment:
Please do not use numbers like `1.602e6 `- but instead define the contents using `scipy.constants`

Reply:
That seems to be the wrong constant: 
```
from scipy import constants
constants.eV*constants.bar*constants.angstrom**3
``` 
equals `1.6021766208000003e-44`

Reply:
`1/constants.eV*constants.bar*constants.angstrom**3`

Reply:
Previously you had `1/1.602e6` now it is `1/1.6021766208000003e-44` - so it seem like there is a factor missing.

Reply:
`1/constants.eV*constants.bar*constants.angstrom**3 = 6.241509125883258e-07`

Reply:
`1/1.602e6 = 6.242197253433208e-07`

Reply:
It is still not the same like `1.602e6` which you had previously.

Reply:
Now I get it. Looks fine to me.